Pod/Classes/Plugin/Core/UICorePlugin.swift

```diff
@@ -16,6 +16,15 @@ public class UICorePlugin: UIPlugin, Plugin {
         super.init(frame: CGRectZero)
     }
 
+    public required init(context: UIBaseObject) {
+        super.init(frame: CGRectZero)
+        if let tempCore = context as? Core {
```

Comment:
Same thing here: use implicitly unwrapped optional and avoid using `temp` prefixes.